assets/app/features/upgrade/templates/landing-page.jade

```diff
@@ -1,39 +1,57 @@
-        li(translate='') upgrade.steps.landing.requirements.point3
+    .row
+        .col-md-6
+            .prechecks.pull-left
+                crowbar-circle.active 1
+                .h4.text-center(translate='') upgrade.steps.landing.prechecks.title
 
-    h2.h4(translate='') upgrade.steps.landing.prechecks.title
-    //- div.checks-container
-    .panel.panel-default
-        .panel-body
-            p(translate='') upgrade.steps.landing.prechecks.description
-            crowbar-checklist(checklist='upgradeLandingVm.prechecks.checks', completed='upgradeLandingVm.prechecks.completed')
+                p(translate='') upgrade.steps.landing.prechecks.description
+                crowbar-checklist(checklist='upgradeLandingVm.prechecks.checks', completed='upgradeLandingVm.prechecks.completed')
 
-            button.btn.btn-success(
-                ng-disabled="(upgradeLandingVm.prechecks.completed && upgradeLandingVm.prechecks.valid) || upgradeLandingVm.prechecks.running",
-                ng-class="{\
-                    'spinner-visible': upgradeLandingVm.prechecks.spinnerVisible,\
-                    active: upgradeLandingVm.prechecks.running\
-                }",
-                ng-click="upgradeLandingVm.prechecks.runPrechecks()"
-            )
-                suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prechecks.running",
+                button.btn.btn-primary(
+                    ng-disabled="upgradeLandingVm.prechecks.running",
+                    ng-class="{\
+                            'spinner-visible': upgradeLandingVm.prechecks.spinnerVisible,\
+                            active: upgradeLandingVm.prechecks.running\
+                        }",
+                    ng-click="upgradeLandingVm.prechecks.runPrechecks()"
+                )
+                    suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prechecks.running",
                     visible="upgradeLandingVm.prechecks.spinnerVisible")
-                span(translate='') upgrade.steps.landing.prechecks.form.check-again
-
-    .row
-        .col-sm-10.col-sm-offset-1.note-message.text-center
-            strong(translate='') upgrade.steps.landing.note
+                    span(translate='') upgrade.steps.landing.prechecks.form.check-again
 
-    button.btn.btn-success.center-block(
-        ng-disabled="!upgradeLandingVm.prechecks.completed || !upgradeLandingVm.prechecks.valid || upgradeLandingVm.prepare.running",
-        ng-class="{'spinner-visible': upgradeLandingVm.prepare.spinnerVisible, active: upgradeLandingVm.prepare.running}",
-        ng-click="upgradeLandingVm.beginUpgrade()"
-    )
-        suse-lazy-spinner(delay="2000", active="upgradeLandingVm.prepare.running",
-            visible="upgradeLandingVm.prepare.spinnerVisible")
-        span(translate='') upgrade.steps.landing.form.button
+            .upgrade.pull-left(ng-class="{'active': upgradeLandingVm.mode.valid, 'inactive': !upgradeLandingVm.mode.valid}")
```

Comment:
actually I only needed the active class here it seems. I think is a leftover from before.